Vagrantfile

```diff
@@ -78,9 +78,10 @@ systemctl start mysql
 
 cd /vagrant
 bundle config build.nokogiri --use-system-libraries
-bundle install
+bundle install --retry=3
 bundle exec rake db:create
 bundle exec rake db:migrate
+bundle exec rake portus:create
```

Comment:
Shouldn't we run `db:seed instead? I foresee one day we will remove this special task.